playbooks/common/openshift-cluster/upgrades/library/openshift_upgrade_config.py

```diff
@@ -78,6 +78,10 @@ def upgrade_master_3_0_to_3_1(ansible_module, config_base, backup):
         config['kubernetesMasterConfig'].pop('apiLevels')
         changes.append('master-config.yaml: removed kubernetesMasterConfig.apiLevels')
 
+    # Add masterCA to serviceAccountConfig
+    if 'serviceAccountConfig' in config and 'masterCA' not in config['serviceAccountConfig']:
+        config['serviceAccountConfig']['masterCA'] = 'ca.crt'
```

Comment:
@brenton good point, we currently hardcode all references to ca.crt, but I agree we should not assume. I'll update.